lib/react_on_rails/git_utils.rb

```diff
@@ -4,7 +4,7 @@
 
 module ReactOnRails
   module GitUtils
-    def self.uncommitted_changes?(message_handler, git_installed = true)
```

Comment:
Do only tests use this? if so, maybe we shouldn't have the option?

I'm guessing all other usages assume git is installed, and if so, then we don't need the option.

In other words, does React on Rails break if git is not installed?